stdlib/tests/crypto/rpo.rs

```diff
@@ -354,16 +333,50 @@ fn test_hash_memory() {
 
 #[test]
 fn test_hash_memory_empty() {
-    // try to hash 0 values
-    let compute_inputs_hash = "
+    // absorb_double_words_from_memory
+    let source = "
+    use.std::crypto::hashes::rpo
+
+    begin
+        push.1000      # end address
+        push.1000      # start address
+        padw padw padw # hasher state
+
+        exec.rpo::absorb_double_words_from_memory
+    end
+    ";
+
+    let mut expected_stack = vec![0; 12];
+    expected_stack.push(1000);
+    expected_stack.push(1000);
+
+    build_test!(source, &[]).expect_stack(&expected_stack);
+
+    // hash_memory_words
+    let source = "
+    use.std::crypto::hashes::rpo
+
+    begin
+        push.1000 # end address
+        push.1000 # start address
+
+        exec.rpo::hash_memory_words
+    end
+    ";
+
+    build_test!(source, &[]).expect_stack(&[0; 4]);
```

Comment:
Why not test against expected stack `[0; 16]`?

Reply:
Let's check the entire stack - this way we'll be sure that the procedure doesn't "mess" with the rest of the stack.